baybe/searchspace/core.py

```diff
@@ -223,17 +224,11 @@ def type(self) -> SearchSpaceType:
         raise RuntimeError("This line should be impossible to reach.")
 
     @property
-    def contains_mordred(self) -> bool:
-        """Indicates if any of the discrete parameters uses ``MORDRED`` encoding."""
+    def contains_fingerprint(self) -> bool:
```

Comment:
hmm agree seems strange, I guess this function should simply check whether there is a subtance parameter with either mordred or rdkit2ddescriptors?

Reply:
That would at least keep it consistent to what is was designed for and would warrant moving it to the edbo file itself. I'd be happy with that change 👍🏼 I think the open question is only how we want the edbo logic to behave in case of other descriptors. Since edbo is not defined for this case, simply using the already existing fallback options implemented in that file is probably the most sensible thing to do and would automatically happen with your suggested change. If you agree, I can take care of it

Reply:
Have changed it. If happy, simply resolve

Reply:
ah, good point, forgot that `RDKIT` is the the deprecated one. So no, it doesn't. Simply adding `RDKIT2DDESCRIPTORS` should be fine, right?

Reply:
i though it needs to be both, checked here is the attribute of the parameter, cant that be both since the deprecation changes would happen somewhere else downsstream?

Reply:
Yes exactly. The alternative that would avoid that is if we apply the deprecation mechanism while setting the encoding value, i.e. via a temporary converter. But I don't think that's necessary. So let me just add the member to the current list and you can confirm by resolving.

Reply:
done

Reply:
Is this for @Scienfitz  or me to resolve?

Reply:
seems good, resolving